include-what-you-use.1

```diff
@@ -38,6 +38,11 @@ All other options are interpreted as
 .BR clang (1)
 compiler options.
 .TP
+.B \-\-canonical_stdlib
```

Comment:
I'm a bit skeptical to the whole "canonical" nomenclature. Is it more like `--std_symbols`? Not great either.